### PR TITLE
The overlay shouldn't glitch out if there is no title provided

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
This helps solve some of the strange behaviour we were seeing with overlays on @CodeBaboon's computer (Turns out that something was leaving his overlay without a title)

### CODE REVIEW

File: d2l-simple-overlay-styles.html [hunk 1]
```diff
@@ -26,11 +26,19 @@
...
+			top: 0;
+			bottom: 0;
+			margin-top: auto;
+			margin-bottom: auto;
+			position: absolute;
+			right: 0;
+		}
+
+		.container {
+			position: relative;
```
Comment (ryantmer) [inline d2l-simple-overlay-styles.html]: Could this same result be achieved (in perhaps a simpler way) using `display: flex` along with `justify-content: space-between`?

Comment (jstefaniuk-d2l) [inline d2l-simple-overlay-styles.html]: Probably, will try!

File: d2l-simple-overlay-styles.html [hunk 2]
```diff
@@ -26,19 +26,17 @@
...
-			right: 0;
 		}
 
 		.container {
-			position: relative;
+			display: flex;
+			justify-content: space-between;
+			align-items: center;
+			min-height: 105px;
+			flex-direction: row-reverse;
```
Comment (ryantmer) [inline d2l-simple-overlay-styles.html]: Could also just swap the order in HTML, I think? (I've never actually used `row-reverse` before though, so if it does something else as well, then neat!)

**Comment by jstefaniuk-d2l** [comment]
@ryantmer good to merge?

**Comment by ryantmer** [comment]
:+1:

**Pull request merged**